modules/space/rendering/renderableorbitalkepler.cpp

```diff
@@ -562,7 +565,9 @@ void RenderableOrbitalKepler::render(const RenderData& data, RendererTasks&) {
         const float fade = pow(
             _appearance.trailFade.maxValue() - _appearance.trailFade, 2.f
         );
-        _trailProgram->setUniform(_uniformTrailCache.trailFade, fade);
+        _trailProgram->setUniform(_uniformTrailCache.trailFadeExponent, fade);
+
+        _trailProgram->setUniform(_uniformTrailCache.colorFadeCutoffPoint, 0.05f);
```

Comment:
Maybe add a comment here why it is 0.05?  The uniform is never set to another value either, so maybe just declare it as a constant in the shader instead?

Reply:
The cutoff value is being used in both geometry and fragment shader for trails.
The reason I made it an uniform was to minimize the risk if it being different in each shader.
I'll add a comment explaining 0.05.

Reply:
>0.05 is the "alpha value" for which the trail should no longer be rendered.
>The value that's compared to 0.05 is calculated in the shader and depends
>on the distance from the head of the trail to the part that's being rendered.
>Value is passed as uniform due to it being used in both geometry and fragment
>shader.

Too descriptive?

Reply:
I think that's great :+1: